spec/features/users/oidc_login_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+describe "OpenId Connect Login", type: :feature do
+  context "with valid credentials" do
+    before do
+      OmniAuth.config.mock_auth[:openid_connect] = OmniAuth::AuthHash.new(
+        provider: "openid_connect",
+        uid: "123456789",
+        info: {
+          email: "test@test.com"
+        }
+      )
+
+      visit new_user_session_path
+      find_by_id('openid_connect-signin').click
+    end
+
+    it "redirects to root path" do
+      expect(page).to have_current_path(root_path)
+    end
+
+    it "displays a success message" do
+      expect(page).to have_css(".alert-success")
+    end
+  end
+
+  context "with invalid credentials" do
+    before do
+      @omniauth_logger = OmniAuth.config.logger
+      # Change OmniAuth logger (default output to STDOUT)
```

Comment:
I'm assuming the idea here is to hide the OmniAuth log output. Maybe we could keep it inplace for now, and remove it if it becomes a big pain in the future? By muting the logs here, I'm fearing that it could be harder to debug test failures, e.g. in CI.

Reply:
agreed and resolved.